src/Stratis.Bitcoin.IntegrationTests/PoSMintingTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System.Linq;
+using NBitcoin;
+using Stratis.Bitcoin.Features.Miner.Interfaces;
+using Stratis.Bitcoin.Features.Wallet;
+using Stratis.Bitcoin.Features.Wallet.Controllers;
+using Stratis.Bitcoin.Features.Wallet.Models;
+using Stratis.Bitcoin.IntegrationTests.EnvironmentMockUpHelpers;
+using Xunit;
+using static Stratis.Bitcoin.Features.Miner.PosMinting;
+
+namespace Stratis.Bitcoin.IntegrationTests
+{
+    public sealed class PoSMintingTests
+    {
+        [Fact]
+        public void ProofOfStake_MintNewCoinsViaStaking()
+        {
+            using (NodeBuilder builder = NodeBuilder.Create())
+            {
+                var nodeGenerateCoins = builder.SkipRules().CreateStratisPowNode();
```

Comment:
This almost defeats the purpose of the test no?

Reply:
No, see comment above, this is is to quickly generate x amount of blocks. Its either this or we have to use the now obsolete `GenerateStratis` method.

Reply:
If I understand correctly you are removing all the rules apart for one.